main.py

```diff
@@ -1 +1,107 @@
-x = 50
+import pandas as pd
+import pyodbc
+
+def database_connect(server, database):
+    try:
+        conn = pyodbc.connect(
+        Trusted_Connection='Yes',
+        Driver='{ODBC Driver 17 for SQL Server}',
+        Server=server,
+        Database=database
+        )
+        cursor = conn.cursor()
+        return conn, cursor
+
+    except Exception as e:
```

Comment:
It is better if you do not use a bare Exception here, either check how the try block can fail (and use that exception), or write a custom exception as follows:

class DatabaseConnectionError(Exception):
    pass

Reply:
Can you give me an example? Thanks!

Reply:
something like this, but with a better print statement (maybe returning the parameters you introduced in the print statement):
https://stackoverflow.com/a/34312912